_includes/project-new.md

```diff
@@ -96,7 +98,7 @@ sequencing type, select `Other`.
 
 ## 2.2 My Projects tab
 
-1. Click again on the _My Projects_ tab.
+1. Click _My Projects_ in the _Tracking_ section.
```

Comment:
I'm not sure this is accurate, as _My Projects_ is a tab. I don't believe there's a _My Projects_ link in the _Tracking_ section, but are you seeing something different?